iocore/net/SSLUtils.cc

```diff
@@ -1990,6 +1990,12 @@ SSLInitServerContext(const SSLConfigParams *params, const ssl_user_config *sslMu
   if (SSLConfigParams::ssl_ocsp_enabled) {
     Debug("ssl", "SSL OCSP Stapling is enabled");
     SSL_CTX_set_tlsext_status_cb(ctx, ssl_callback_ocsp_stapling);
+    const char *certname = sslMultCertSettings->cert.get();
+    for (auto cert : cert_list) {
+      if (!ssl_stapling_init_cert(ctx, cert, certname)) {
```

Comment:
Should this be called in `ssl_store_ssl_context()` by any chance?

Reply:
Yes, I agree with your analysis that doing all the ocsp initialization in SSLInitServrerContext makes sense.  When the SSLServerCreateContext was made, some of the ocsp logic was pulled imto SSLInitServerContext so it would be enabled in the InkAPI call too.